### DISCUSSION TITLE
Is it possible to reduce install weight by only including one locale file: https://pdfviewer.net/extended-pdf-viewer/i18n

### DISCUSSION THREAD

**Discussion by ICEDLEE337** [stephanrauh/ngx-extended-pdf-viewer#1297] [General]
I followed the steps defined at https://pdfviewer.net/extended-pdf-viewer/i18n to only include `en-US` which basically involves:
1. [useBrowserLocale]="false"
2. [language]="'en-US'"
3. <script type="application/l10n">{"default_locale":"en-US","locales":{"en-US":"blah blah blah...."}}</script> added to index.html

However if I change from [language]="'en-US'" to [language]="'ja'" then I notice that Japanese is rendered even though I didn't make that language snippet available which leads me to believe I'm not getting any reduction in bundle size.

How can I make the additional locale files unavailable to reduce bundle size? Thank you!

**Comment by ICEDLEE337** [comment]
I just saw another question that answers this question. Basically I need to adjust the angular.json file as well (duh!). Thanks.

**Reply by stephanrauh** [reply]
I'm confused by the Japanese bit. Can you check the network traffic? Does the viewer load the `locale.properties` and the `viewer.properties`? If it does, you've found a bug!
